### PR TITLE
4626

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: docs/CREATE4646.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# 4626 Market Creation
+
+In order to create a market, you must first have a market creator account. This account will be funded with an initial mint amount to establish the share price of the market.
+
+## Deployment Steps
+
+1. Set REWARDS_RECEIVER to the address you want to receive rewards in Addresses.sol on the network you're deploying on.
+2. Fund the deployer with the following token amounts:
```
Comment (lyoungblood) [inline docs/CREATE4646.md:8]: ~~What's the reason for deployer to need tokens? Is it to test deposits?~~

Nvm, found it below.

File: test/proposals/Deploy4626Vaults.s.sol [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                addresses.getAddress("MOONWELL_USDC")
+            );
+            assertEq(address(vault.comptroller()), unitroller);
+            assertEq(vault.rewardRecipient(), rewardRecipient);
+
+            console.log("deployed USDC vault: ", address(vault));
+
+            ERC20 usdc = ERC20(addresses.getAddress("USDC"));
+            usdc.approve(address(vault), INITIAL_MINT_AMOUNT_USDC);
+            vault.deposit(INITIAL_MINT_AMOUNT_USDC, deployerAddress);
```
Comment (lyoungblood) [inline test/proposals/Deploy4626Vaults.s.sol:90]: Ah, here it is. Nvm about the first comment.

File: test/proposals/mips/mip-b00/mip-b00.sol
Comment (lyoungblood) [inline test/proposals/mips/mip-b00/mip-b00.sol:1]: Did you intend to have this change in the PR?

**Review by lyoungblood — COMMENTED** [review]
Left comments.

File: test/proposals/mips/mip-b00/mip-b00.sol
Comment (ElliotFriedman) [reply test/proposals/mips/mip-b00/mip-b00.sol:1]: yes

**Review by lyoungblood — APPROVED** [review]

**Pull request merged**